WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -140,6 +140,22 @@ def records
     render json: json
   end
 
+  def export_public
+    sql_zips = Dir.glob(Rails.root.join("../webroot/results/misc/*.sql.zip")).sort!
```

Comment:
How many of these do we have, again? It seems to return at least as fast as other HTTP requests, so I'm not super concerned.

Reply:
I think we have at most 2 of these, as the PHP code deletes the old export once it's done generating the next export: https://github.com/thewca/worldcubeassociation.org/blob/00b66534978cd467eff5c0ecef0e12971deb2bcf/webroot/results/admin/export_public.php#L232-L235.